sdks/python/run_postcommit.sh

```diff
@@ -54,29 +54,43 @@ echo ">>> RUNNING DIRECT RUNNER py-wordcount"
 python -m apache_beam.examples.wordcount --output /tmp/py-wordcount-direct
 # TODO: check that output file is generated for Direct Runner.
 
-# Run wordcount on the service.
+# Run tests on the service.
 
 # Where to store wordcount output.
 GCS_LOCATION=gs://temp-storage-for-end-to-end-tests
 
 # Job name needs to be unique
-JOBNAME=py-wordcount-`date +%s`
+JOBNAME_E2E=py-wordcount-`date +%s`
+JOBNAME_VR_TEST=py-validatesrunner-`date +%s`
 
 PROJECT=apache-beam-testing
 
 # Create a tarball
 python setup.py sdist
 
+SDK_LOCATION=$(find dist/apache-beam-sdk-*.tar.gz)
+
+# Run ValidatesRunner tests on Google Cloud Dataflow service
+python setup.py nosetests \
+  -a ValidatesRunner --processes=4 --process-timeout=360 \
+  --test-pipeline-options=" \
+    --runner=BlockingDataflowPipelineRunner \
+    --project=$PROJECT \
+    --staging_location=$GCS_LOCATION/staging-validatesrunner-test \
+    --sdk_location=$SDK_LOCATION \
+    --job_name=$JOBNAME_VR_TEST\
+    --num_workers=1"
```

Comment:
Make sure the job output appears in the logs, it makes debugging easier (see `cat job_output` below`)